monotextor/Snakefile

```diff
@@ -880,41 +879,34 @@ rule monocleaner:
                 | pigz -c > {output}
         """
 
-
-filter_input = rules.monocleaner.output
-if not MONOCLEANER:
-    filter_input = rules.monocleaner.input.monofixer
-
-
+# TODO is correct to apply deduplication in the rule filter? If so, there is no difference between rule raw and sents when PARAGRAPHS="" (i.e. the output files of the rules raw an sents will have the exact same content)
```

Comment:
Yes, it is correct to apply dedup here. Regarding the difference of the rules, they seem to be exact the same shell code. But sents produces a deduped and filtered output, and raw produces unfiltered and undeduped. Inputs are not the same.

Reply:
Ah, ok! When I was making tests I noticed that produced the same files when paragraphs were disabled. I guess that was coincidence that there were not duplicate sentences.